.gitignore

```diff
@@ -8,3 +8,4 @@ index.cjs.js
 index.esm.js
 index.cjs.js.map
 index.esm.js.map
+*.swp
```

Comment:
@brekk please remove this and add `index.umd.js` and `index.umd.js.map` to ignored files